### DISCUSSION TITLE
Possibility to check photos (subsequent checking, for example, for privacy reasons before sending)

### DISCUSSION THREAD

**Discussion by OpenGreenStreet** [streetcomplete/StreetComplete#2394] [Ideas]
Possibility to check photos (subsequent checking, for example, for privacy reasons before sending)
(de: Möglichkeit zum Prüfen von Fotos (nachträgliches Prüfen zum Beispiel aus Gründen des Datenschutz vor dem Versenden)

**Comment by matkoniecz** [comment]
Is it depending on a phone/Android version? My Xiaomi A2 Lite with Android 10 is showing me full screen view of a photo after taking it and allows me to accept/reject it.
